Add MO command to PyTorch conversion documentation (#9487)
openvinotoolkit/openvino: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md

torch.onnx.export(model, (dummy_input, ), 'model.onnx')
```

The ONNX model can be loaded in OpenVINO Inference Engine directly, without converting the model to IR format. See the [ONNX support](https://docs.openvino.ai/latest/openvino_docs_IE_DG_ONNX_Support.html) documentation.

## Export ONNX Model to OpenVINO <a name="export-to-openvino"></a>

To export an ONNX Model to OpenVINO, you can use Model Optimizer. If you installed the
[openvino-dev](https://pypi.org/project/openvino-dev/) pip package, the command is:

```shell
mo --input_model model.onnx
```

Replace `mo` with `python /path/to/model_optimizer.py` or `python3 /path/to/model_optimizer.py` for other OpenVINO installations.

See [Convert an ONNX model](Convert_Model_From_ONNX.md) for more information and advanced options.

## Known Issues

> * Not all PyTorch operations can be exported to ONNX opset 9 which is used by default, as of version 1.8.1.

Reviewer: ```suggestion
* Not all PyTorch operations can be exported to ONNX opset 9, which, as of version 1.8.1., is used by default.
```

> It is recommended to export models to opset 11 or higher when export to default opset 9 is not working. In that case, use `opset_version`

Reviewer: ```suggestion
It is recommended to export models to opset 11 or higher when export to the default opset 9 does not work. In that case, use the `opset_version`
```

> option of the `torch.onnx.export`. For more information about ONNX* opset, refer to the [Operator Schemas](https://github.com/onnx/onnx/blob/master/docs/Operators.md).

Reviewer: ```suggestion
option of `torch.onnx.export`. For more information about ONNX opset, refer to the [Operator Schemas](https://github.com/onnx/onnx/blob/master/docs/Operators.md).
```
